src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-baseline-alert-conditions.mdx

```diff
@@ -46,9 +46,9 @@ When your data escapes the predicted "normal" behavior and meets the criteria yo
 
 To create a baseline condition: When you start to create a [condition](/docs/alerts/new-relic-alerts/defining-conditions/define-alert-conditions#create-condition), choose one of the following [data sources](#availability):
```

Comment:
I think this screen shot should be updated to not be a faceted query.  Here is one.  
![image](https://user-images.githubusercontent.com/26093145/116591393-9c5f7700-a8d3-11eb-8c22-3a17747e9f08.png)


Additionally, I think we should add a section indicating that faceted baseline queries are now (newly?) supported.  Something like:

Faceted baseline conditions are now supported. 

![image](https://user-images.githubusercontent.com/26093145/116591505-bf8a2680-a8d3-11eb-85f6-d0685ddb06f3.png)

After inputting a faceted baseline queries, you can scope your results to a single baseline by clicking on a time series in the chart legend.  From here, you will see the baseline, threshold band, and violation region(s) if there are any for this baseline.  You can return to see all the results by clicking "Return to full time series list."

![image](https://user-images.githubusercontent.com/26093145/116591726-09730c80-a8d4-11eb-9c76-22ec527ef249.png)